src/features/bundlr/BundlrProfile.tsx

```diff
@@ -87,18 +84,21 @@ export function BundlrProfile({ action, onDeposit, onWithdraw, onConnect }: Bund
           You have a pending balance update. Your balance should be updated in a few minutes.
         </Text>
       )}
-      <Text mt={6}>Enter Amount</Text>
-      <BundlrInput
-        value={inputAmount}
-        onChange={handleChangeAmount}
-      />
+      <Text
+        mt={6}
+        mb={3}
+      >
+        Enter Amount
+      </Text>
+      {BundlrInput}
```

Comment:
With the change made in a previous comment this can be changed to `<BundlrInput />`